dev/ci/ci-bedrock2.sh

```diff
@@ -6,4 +6,4 @@ ci_dir="$(dirname "$0")"
 FORCE_GIT=1
 git_download bedrock2
 
-( cd "${CI_BUILD_DIR}/bedrock2" && git submodule update --init --recursive && COQMF_ARGS='-arg "-async-proofs-tac-j 1"' make && make install )
+( cd "${CI_BUILD_DIR}/bedrock2" && git submodule update --init --recursive && COQEXTRAFLAGS='-native-compiler no' COQMF_ARGS='-arg "-async-proofs-tac-j 1"' make && make install )
```

Comment:
I think I'd prefer to do an `export COQEXTRAFLAGS=...` before the subshell, the command is getting kinda heavy.
(same for the other modified scripts)